react/react1/week1/static-todo-list/src/components/Title.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Title = () => {
+  return (
+    <div>
+      <div>n.</div>
```

Comment:
I don't understand why you see it.. because my Title.js component looks different)
![image](https://user-images.githubusercontent.com/73549460/193463916-09d41547-e203-4d97-bd35-2d003fc2e60d.png)